cli/ssync/main.go

```diff
@@ -33,16 +34,17 @@ Examples:
 
 	args := flag.Args()
 	if *daemon {
-		// Daemon mode
-		endpoint := sparse.TCPEndPoint{Host: "" /*bind to all*/, Port: int16(*port)}
 		if *verbose {
 			log.SetLevel(defaultVerboseLogLevel)
-			fmt.Fprintln(os.Stderr, "Listening on", endpoint, "...")
 		} else {
 			log.SetLevel(defaultNonVerboseLogLevel)
 		}
 
-		sparse.Server(endpoint, *timeout)
+		err := rest.Server(*port)
+		if err != nil {
+			log.Fatalf("Ssync server is not started up correctly, err: %s", err)
+			os.Exit(1)
```

Comment:
log.Fatalf() will call os.Exit(1) and result in process exit. Don't need this line.